app/src/main/java/com/teamltt/carcare/activity/HomeActivity.java

```diff
@@ -193,4 +188,36 @@ public void toggleLogging(MenuItem item) {
         }
 
     }
+
+    public void displayStaticData() {
+        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(this);
+        String syncConnPref = preferences.getString(SettingsActivity.KEY_PREF_SYNC_CONN, "");
+
+        TextView tvStaticDataName = ((TextView) findViewById(R.id.tvStaticDataName));
+        TextView tvStaticDataValue = ((TextView) findViewById(R.id.tvStaticDataValue));
+
+        String tvTextName = "";
+        String tvTextValue = "";
+
+        if (preferences.getBoolean("sEngineTemp", false)) {
+            //TODO Read from database
+            tvTextName +="Engine Temperature:";
+            tvTextValue += "80 F ";
+        }
+
+        if (preferences.getBoolean("sMPG", false)) {
+            //TODO Read from database
+            tvTextName += "\nCurrent Miles Per Gallon:";
```

Comment:
I don't know if you have any more knowledge than I do with this, but I currently have

`public void displayStaticData() {
        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(this);
        String syncConnPref = preferences.getString(SettingsActivity.KEY_PREF_SYNC_CONN, "");

        LinearLayout lvStaticData = ((LinearLayout) findViewById(R.id.static_data));
        lvStaticData.removeAllViews();

        boolean haveStatic = false;

        if (preferences.getBoolean("@string/key_engine_temp", false)) {
            lvStaticData.addView(createTextView("Engine Temperature:", 3));
            lvStaticData.addView(createTextView("80 F", 5));
            haveStatic = true;
        }
        if (preferences.getBoolean("@string/key_mpg", false)) {
            lvStaticData.addView(createTextView("Current Miles Per Gallon:", 3));
            lvStaticData.addView(createTextView("35 mpg", 5));
            haveStatic = true;
        }
        if (preferences.getBoolean("@string/key_mph", false)) {
            lvStaticData.addView(createTextView("Current Miles Per Hour:", 3));
            lvStaticData.addView(createTextView("60 mph", 5));
            haveStatic = true;
        }
        if (!haveStatic) {
            findViewById(R.id.static_data_card).setVisibility(View.INVISIBLE);
        } else {
            findViewById(R.id.static_data_card).setVisibility(View.VISIBLE);
        }
    }`
and it's either showing only the engine temperature text views or nothing except the cardview.